### PR TITLE
Marthinus

### PR DESCRIPTION
**Pull request by marthinus-tmp**
# Added NavUp-Visualize

-use case diagram has a TODO
-only a single large use case diagram (not sure if that is correct)
-format is different to master.
-might have missed stuff
-hope it is up to par

### CODE REVIEW

File: NavUPFunctionalRequirements.tex [hunk 1]
```diff
@@ -40,17 +40,48 @@
 	\section{Overall Description}
 		\subsection{Product Functions}
 		% Provide a summary of the functions of the product.
+
+%BEGIN NavUp-Visualize
+%at the correct place?
+		\paragraph{}
```
Comment (AndreasNel) [inline NavUPFunctionalRequirements.tex]: Give titles for your paragraph, but if you prefer not to have a title, take out the ```\paragraph``` tags and just type as you normally would.

File: Raw_NavUp-Visual.txt [hunk 2]
```diff
@@ -0,0 +1,12 @@
+% Product Functions
```
Comment (AndreasNel) [inline Raw_NavUp-Visual.txt]: Remove this text file.

File: NavUp-Visualize.xml [hunk 3]
```diff
@@ -0,0 +1 @@
...YL9Jtu4Nkr8NCqxBicXaRVI5RG38VMVkxc95w4XR4014+9pwaMifV0GGRjoUexl9Grph1UKAjisn9CbbQwYVx6TvAbegB2JoaTkmTfKUkNpQm2QoHB3Y/mnSi2prltQBsjERNX4yQODIbKDqw9trAUUs9trZMeAYy7XlKyBtaB3DmCkrU1aaRMjVHhTUYN1Nrqd0eCq89gGrN1MaDl/Xe4KWhwjIuQf3PGwdtVs96/wBAAJ3lnwgQA5UOzqlgjQ365jgKVhd0VL/0QdyJ3DjTkSMWOxMYt49TRHc6rqZc/TCbJbBJeT1oq499dNYodX8Qa/RcxbVdyGkmWpJVFsHAfH7pGVXVwBpmz8BSprX3TjstpZTEvIa+TzDgt/nKDO2cRqnu/5jWKNtSuUGRT9+70FW0HosKSZXWba8wpLOVrdVIqaZRfU3Jcn8z0YjcR9//co+ZsmsR692ZyKTVDnB8Mno+E3Hfm4lYntlEtj8fGm3c/RONxB6tU7NHPRkVjPExIUuEz7sZRF1eMN0JLc9uS8u6GWOWjr0tb886a/da7TBLbSs/FtDYxyStJjQn3dhnOx285dg12r7B7kRM0dbTuT8WdW3pi91RbOhC3YXkkFMFkJFgp24FPDfs9PQ4X+rnRCygwSJI9Uggv28IdxjFUb46P0ERwW7KrS0kuKeKCHq6IoSkLes9yc0VfZbu/pwPRdp2R5zsQxF9p+/h3u7AL0XGL+FITPXxgdO4OOnRpErVwndvD2cqE/X0cIdSZNWTyvuMRZGlWOuwYkp+Ek2S0DVZkTjAd1XvdW6UMBLQ6LfL7kC85Q39Ai0VbGZ5/0JKd+LzySCjhHVVb/CFcBdxxE6+VjR3VCR7w7SvQ3AO2739gRxCFxWf6FszTzFkY6BHKNOMXfP5xl5xUXeniCR3NIegU/SH588a1KYvTLV9ptmaHo8Rfp0WxvjtO+t4yhK+8LTOq3jMwTATyPduWkbYxiKrekLJJzHJomkqe0qIKr8OqwuwzbI0ExgkQb2098wFZIQkSfg36kxO2YJxe04kb/jN4oj9xoSfiUpWbkSIj8b7FvemF22Zz+wUHE5RunH0qtb3b7czcMXlA3MRc0Hoe2fPtVG5uIv8xwH8vksSUxGmWZo8ikp8R/EWmkJA2yfzMks/QCGsWf1DgsLDVv/0Adz9Dw==</diagram></mxfile>
```
Comment (AndreasNel) [inline NavUp-Visualize.xml]: You can remove this xml file, draw.io is capable of importing from png.

**Comment by marthinus-tmp** [comment]
# Update
All errors have been corrected (hopefully)

### concerns
-Still not sure with what subsystems to interact, or is it sufficient to interact directly with the NavUp system?

**Pull request merged**